srcs/builtin/builtin_cd.c

```diff
@@ -115,7 +115,7 @@ void		builtin_cd(char **argv, t_environ *env, t_exec *exe)
 	exe->ret = 0;
 	if (!(getcwd(cwd, MAX_ENV_ENTRY_LEN)))
```

Comment:
```suggestion
	if (!(getcwd(cwd, MAX_ENV_ENTRY_LEN)) && errno != ENOENT)
	{
		if (errno == EACCES)
```
Pour fixe le bug qu'a trouvé @Gamouche